packages/react-native/Libraries/StyleSheet/StyleSheetTypes.d.ts

```diff
@@ -274,6 +276,7 @@ export interface ViewStyle extends FlexStyle, ShadowStyleIOS, TransformsStyle {
    * Controls whether the View can be the target of touch events.
    */
   pointerEvents?: 'box-none' | 'none' | 'box-only' | 'auto' | undefined;
+  cursor?: CursorValue;
```

Comment:
When I cherry-picked this to `react-native-visionos` fork, I've noticed this is needed (that's why test-js is failing):

```suggestion
  cursor?: CursorValue | undefined;
```